lib/project_printer.rb

```diff
@@ -1,48 +1,51 @@
+require_relative "font_color_and_style"
+
 class ProjectPrinter
-  attr_accessor :stdout
+  attr_accessor :stdout, :colors
 
   def initialize(stdout)
     @stdout = stdout
+    @colors = FontColorAndStyle.new
   end
 
   def projects(project)
-    stdout.puts("  #{project.keys.first}\n")
+    stdout.puts("  #{project.keys.first}")
   end
 
   def deleting_a_project(project_name)
-    stdout.puts "\e[38;5;40mDeleting project:\e[0m '#{project_name.strip}'\n\n"
+    stdout.puts colors.green + "Deleting project: '#{project_name.strip}'" + colors.escape
```

Comment:
Using string interpolation vs `+` with strings is more rubyesque, and safer. If you add a string to nil you get an error, but if you interpolate in a nil you end up with nothing added. But also the added noise around operators is high. 

I also noticed that you are using puts with parens and without parens. It would be good to get consistent. That kind of consistency means they eye can run down the file and quickly identify the true differences instead of just the noise of different syntax.